controller/backup_volume_controller.go

```diff
@@ -284,11 +284,26 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 		return err
 	}
 
+	// Get `Time to live` after failed backup was marked as `Error` or `Unknown`,
+	// default: 180s and it is shorter than `Backupstore Poll Interval` by default, so failed backups should be deleted once the backup volume is updated.
+	failedBackupTTL, err := bvc.ds.GetSettingAsInt(types.SettingNameFailedBackupTTL)
+	if err != nil {
+		log.WithError(err).Errorf("failed to get %v setting", types.SettingNameFailedBackupTTL)
+		// failedBackupTTL would be -1 if there is an error.
+	}
 	clustersSet := sets.NewString()
 	for _, b := range clusterBackups {
 		// Skip the Backup CR which is created from the local cluster and
 		// the snapshot backup hasn't be completed or pulled from the remote backup target yet
 		if b.Spec.SnapshotName != "" && b.Status.State != longhorn.BackupStateCompleted {
+			if b.Status.State == longhorn.BackupStateError || b.Status.State == longhorn.BackupStateUnknown {
```

Comment:
ref: https://github.com/longhorn/longhorn-manager/blob/c3e9b1c8c7d1043134bfbf2c8ceb25c789935bc0/controller/backup_controller.go#L359-L359

For now, the code uses `BackupStateError` and `BackupStateUnknown` as failed backup.

Reply:
Yes, it is now.